spec/rmagick/draw_spec.rb

```diff
@@ -0,0 +1,157 @@
+class Magick::Draw
+  def self._dummy_img_
+    @@_dummy_img_
+  end
+end
+
+describe Magick::Draw do
+  before(:each) { @draw = Magick::Draw.new }
```

Comment:
I try to avoid `let!`, though I guess this is effectively the same. Let me try `let`, if we prefer that over instance variables.